content/zh/docs/concepts/context-propagation.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 上下文传播
+weight: 10
+description: 了解实现分布式追踪的概念。
+---
+
+<!--
+---
+title: Context Propagation
+weight: 10
+description: Learn about the concept that enables Distributed Tracing.
+---
+-->
```

Comment:
I think it would be easier to review if the original text is annotated here. Otherwise, the reviewer needs to open the original text to compare, which makes the review work more cumbersome. What do you think?

Reply:
Fine, thanks, it can provide some more conveniences.

Reply:
I second @windsonsea comment, this should be removed in the spirit of keeping things [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Otherwise, how do you know that the comment is up-to-date? You'll need to open the original file to double check anyways.

Please remove all embedded HTML comments.

What would be more useful would be to include, in the front matter, a **permalink** to the specific version (including the commit hash) of the `en` page that was used to create this translation, but we can discuss this separately. /cc @open-telemetry/docs-approvers

Reply:
I still stick to my point of view for the following reasons:
1. During the first synchronization, we can easily compare with the original text one-to-one, but during the second or subsequent synchronization, it is difficult for both the submitter and the reviewer to find the differences between the translation and the original text without comments. Adding comments is not for the comparison of the first translation, but for the convenience of comparison during subsequent synchronization. It allows us to more easily locate the differences between the translation and the original text. Otherwise, even if we know that the two articles are different, it is difficult to find the differences if the articles are very long.
2. If we continue to keep the translation consistent with the original text, we should make the subsequent work simpler so that more people can participate in it (I think adding comments can make the subsequent synchronization work simpler)
3. I personally think that the localization of k8s documents is a better example: https://github.com/kubernetes/website/blob/main/content/zh-cn/docs/contribute/localization.md?plain=1

Reply:
IMO, it appears to be an advanced option in future, but shall not be mandatory currently.
K8s only applies this policy to the zh-cn localization and not to other languages.

Can we simply follow standard localization practices at the start?

Reply:
Thank you for providing more context, and the reference to the K8s example.

FYI, I've started two threads of discussion in the hopes of figuring out what "best practices" there might already be in use at the CNCF or elsewhere:

- https://github.com/open-telemetry/opentelemetry.io/discussions/4302#discussioncomment-9616612
- https://cloud-native.slack.com/archives/CUJ6W5TLM/p1717149935431579

Let me respond more to your suggestion to embed `en` page segments as HTML comments in `zh` pages in the thread of https://github.com/open-telemetry/opentelemetry.io/discussions/4302#discussioncomment-9616612.

Reply:
is this blocking for merging this PR?

Reply:
Yes, I'd like to propose that you give me a couple of days to explore alternatives and make a concrete proposal for use in this repo. I've created

- #4582

to track this issue. Here's an excerpt:

> Our support for translations is nascent. We have an opportunity to get it right, or at least good enough, from the start.
> 
> Let me put something together to show you what I have in mind. In the meantime, I'd suggest delaying the merge of other zh page translations.

@svrnm, WDYT?

Reply:
Thanks @chalin 

@my-git9 we discussed this during yesterdays SIG comms meeting and for now we do not want to have the english version inlined. As @chalin stated our support for translation is young and experimental and we do not want to make such a decision right now, as removing the blocks of english translations after the fact is harder than adding them **if** we decide to apply this technique.

See https://github.com/open-telemetry/opentelemetry.io/issues/4582 for more details